cli/utilities/utils.py

```diff
@@ -320,3 +323,52 @@ def set_service_state(node, service_id, state):
     if err:
         return False
     return True
+
+
+def bring_node_offline(node, interface="eth0", timeout=120):
+    """
+    Brings a node offlibe by making network interface down for a defined time
+        Args:
+            node (ceph): Node at which the interface has to be bought down
+            interface (str): Node interface name ens3/eth0 etc.
+            timeout (int): Time duration (in secs) for which network has to
+                           be down
+        Returns (Thread): Thread object, None if failed
+        Example:
+            bring_node_offline(mon_node, installer_node, timout=120)
+    """
+    cmd = f"ifconfig {interface} down;sleep {timeout};ifconfig {interface} up"
+    thread = Thread(target=(lambda: node.exec_command(cmd=cmd, sudo=True)))
+    thread.start()
+
+    if is_node_online(node):
+        log.error(
+            f"{node.hostname} is online even after bringing the n/w interface down"
+        )
+        log.info("Waiting for the interface update command to complete")
+        thread.join()
+        return None
+
+    log.info(f"{node.hostname} is made offline for {timeout} seconds")
+    return thread
+
+
+def is_node_online(node):
+    """
+    Checks whether the given node is online. Uses ping to verify the node status
+    Args:
+        node (ceph): Node to be checked for online/offline status
+
+    Returns (bool): Based on if the node is online or not
+
+    """
+    # Adding a sleep for network interface update to take place
```

Comment:
This function should be in general. Add this sleep in `bring_node_offline` function itself as it's specific to bringing down node.